edm.yaml

```diff
@@ -192,21 +192,41 @@ datatypes :
      - fcc::CaloHit hits // Hits used in the cluster
 
   fcc::TrackState:
-    Description: "Track state at a given point on the track."
+    Description: "Track state at a given point on the track. The track parameters are chosen 
+    to be decribed in the curvilinear frame of the track. With the track parameters 
+    (local1:=0,local2:=0,phi,theta,q/p) defined at the origin of a surface perpendicular to 
+    the track. The local coordinates are zero per defintion. The azimuthal and polar angle
+    can be calculated from the direction of the momentum. The last parameter q/p can be 
+    calculated from the charge and the value of the momentum."
     Author : "C. Bernet, B. Hegner"
     Members:
-     - float location // Location on the track. (Radius?)
-     - float omega // Track curvature in cm.
+     - fcc::Point position // global position specifying the location on the track  
+     - float[3] momentum   // the momentum at position  
+     - float charge        // the charge
+     - float[15] cov       // the covariance matrix describing the covariances of the five 
+       		           // track parameters. Duplicated entries have been left out, which 
+		           // reduces thesize to 15 entries      
+
+  fcc::PerigeeTrackState:
```

Comment:
The PerigeeTrackState members do not fit the TrackState members. 
Can't we use the same class? 
If additional info is needed in PerigeeTrackState, this info could be added to the track directly. 
Or the common information should be handled in a uniform way.